src/App.js

```diff
@@ -35,11 +35,13 @@ const tweetsArray = [
 ];
 
 function App() {
-  return (
-    <div className="App">
-      <Tweet />
-    </div>
-  );
+	return (
```

Comment:
Buen trabajo Miguel, aun que imagina que tengamos 100 elementos en nuestro array esto ya no seria rentable en su lugar podemos hacer uso del map para iterar el array y pasamos el i en el map para agregarlo en el componente mas 'padre' dentro d nuestro map para evitarnos el warning 😉 
```suggestion
return (
    <div className="App">
      {tweetsArray.map((tweet, index) => (
        <Tweet key={index} tweet={tweet} />
      ))}
    </div>
  );```